store/queries/candidate_actors.sql

```diff
@@ -25,9 +25,32 @@ WHERE
     did = sqlc.arg(did)
 RETURNING *;
 
+-- name: UpdateActorProfile :exec
+WITH ap as (
+    INSERT INTO actor_profiles
+        (actor_did, id, created_at, indexed_at, display_name, description)
+    VALUES
+        (sqlc.arg(did), sqlc.arg(id), sqlc.arg(updated_at), sqlc.arg(indexed_at), sqlc.arg(display_name), sqlc.arg(description))
```

Comment:
Inconsistency between `created_at` in the db schema and `updated_at` in the generated Go binding - I'd rather use the same in each place. I think since these are historical entries - `created_at` makes more sense than `updated_at`.